arch/arm64/core/cortex_r/arm_mpu.c

```diff
@@ -0,0 +1,490 @@
+/*
+ * Copyright (c) 2017 Linaro Limited.
+ * Copyright (c) 2021 Arm Limited (or its affiliates). All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <init.h>
+#include <kernel.h>
+#include <soc.h>
+#include <arch/arm64/mm.h>
+#include <linker/linker-defs.h>
+
+#define LOG_LEVEL CONFIG_MPU_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(mpu);
```

Comment:
Hint: the log level can go to this macro as a parameter

Reply:
Thanks, I will fix this.